cmd/sync.go

```diff
@@ -562,6 +569,7 @@ func init() {
 
 	rootCmd.AddCommand(syncCmd)
 	syncCmd.PersistentFlags().BoolVar(&raw.recursive, "recursive", true, "True by default, look into sub-directories recursively when syncing between directories. (default true).")
+	syncCmd.PersistentFlags().BoolVar(&raw.preserveNTFSACLs, "preserve-ntfs-acls", false, "False by default. Preserves NTFS ACLs between aware resources (Windows and Azure Files at the moment)")
```

Comment:
Probably best not to say "at the moment" in user facing text, since it will make them wonder if/when something is going to change.